### ISSUE TITLE
Why is this being calculated if its not used?

### ISSUE THREAD

**Issue by kareemkudus** [IMICSLab/pLGG_Predict_Status#3]
https://github.com/IMICSLab/pLGG_Predict_Status/blob/2b7e3e1ec9bb45ab368378c038d6843cd84926b5/main-resnet.py#L468

**Comment by justin13601** [comment]
Yep will comment that out, I wanted to compare the 2 different ways to calculate loss
